src/JackTrip.cpp

```diff
@@ -159,14 +159,14 @@ void JackTrip::setupAudio(
 #ifdef WAIRTOHUB // WAIR
         //Set our Jack client name if we're a hub server or a custom name hasn't been set
 	      if ( mPeerAddress.toStdString() != "" &&
-	          (mJackClientName == gJackDefaultClientName || mJackTripMode == SERVERPINGSERVER)) {
-            mJackClientName = QString(mPeerAddress).replace(":", ".").toLatin1().constData();
+	          (mJackClientName.constData() == gJackDefaultClientName.constData() || mJackTripMode == SERVERPINGSERVER)) {
+	          mJackClientName = QString(mPeerAddress).replace(":", ".");
```

Comment:
I tried to match mixed whitespace here (tab + spaces). Should I fix this block to use spaces only?

Reply:
Yeah, I think all spaces is the way to go. And I don't think a pointer comparison will work here to allow JackTrip as a name because I don't think they're likely to return a pointer to the same memory address.

Reply:
I tested and it seemed to work... when that comparison is true, the client name is replaced by the host name, and that seems to work. If we set `mJackClientName` [here](https://github.com/jacktrip/jacktrip/blob/7e7309cc15a41c54defd06803de2ee83e7cf8ecc/src/JackTrip.cpp#L109) wouldn't it point to the same data?

Reply:
If it works, it works. I thought the QString constructor made a copy of the internal constData that's returned, but it seems like it doesn't.